modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useRef } from 'react';
+import { Box } from '@elementor/ui';
+import PropTypes from 'prop-types';
+
+const THUMBNAIL_SIZE = 64;
+
+export const Thumbnail = ( props ) => {
+	const previewRef = useRef( null );
+
+	useEffect( () => {
+		if ( previewRef.current ) {
+			const previewRoot = previewRef.current.firstElementChild;
```

Comment:
why aren't you using the `previewRef.current` itself?

Reply:
Too long